hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/RingbufferContainer.java

```diff
@@ -103,14 +105,11 @@ public RingbufferContainer(ObjectNamespace namespace, int partitionId) {
      * Constructs a fully initialized ring buffer that can be used immediately. References to other services, the ring buffer
      * store, expiration policy and the config are all set.
      *
-     * @param namespace            the namespace of the ring buffer container
-     * @param config               the configuration of the ring buffer
-     * @param serializationService the serialization service
-     * @param configClassLoader    the class loader for which the ring buffer store classes will be loaded
```

Comment:
We never used a different classloader than the one from the `NodeEngine`. Since we need the `NodeEngine` internally now, we can drop this parameter.